diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..735c2fe 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -317,7 +317,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
 
     bool success = transferStrategy.performTransfer(to, reward, amount);
 
-    require(success == true, 'TRANSFER_ERROR');
+
   }
 
   /**
